diff --git a/lua/nvchad_ui/statusline/default.lua b/lua/nvchad_ui/statusline/default.lua
index e61cf69..fed7d7d 100644
--- a/lua/nvchad_ui/statusline/default.lua
+++ b/lua/nvchad_ui/statusline/default.lua
@@ -155,10 +155,11 @@ M.cursor_position = function()
   local total_line = fn.line "$"
   local text = math.modf((current_line / total_line) * 100) .. tostring "%%"
 
+  local line, col = unpack(vim.api.nvim_win_get_cursor(0))
   text = (current_line == 1 and "Top") or text
   text = (current_line == total_line and "Bot") or text
 
-  return left_sep .. "%#St_pos_text#" .. " " .. text .. " "
+  return left_sep .. "%#St_pos_text#" .. " " .. line .. "," .. (col+1) .. " " .. text .. " "
 end
 
 M.run = function()
